### ISSUE TITLE
Add theme feature

### ISSUE THREAD

**Issue by thakurthegr8** [thakurthegr8/loop#5]
User should be able to switch themes

**Comment by chayansurana3** [comment]
By theme do you mean dark mode and light mode? If so, I can definitely work on this issue. Please assign this issue to me.

**Comment by thakurthegr8** [comment]
Yes @chayansurana3 but In production it's failing try deploying it at your end to get your pr merged.

**Comment by chayansurana3** [comment]
Hello  @thakurthegr8 ,
I've been working on the project and encountered an issue related to the Firebase configuration setup. Despite following the steps outlined in the contribution.md file, I am consistently facing the auth/invalid-api-key error.

I've double-checked the environment variables in the .env file and ensured that they match the configuration setup as per the contributing.md. However, the issue persists, and I believe there might be an underlying problem with the Firebase API key or project settings.

Could you please provide some guidance or insight on how to troubleshoot this further? Additionally, if there are any specific details or steps that I might have overlooked, please let me know.
Your assistance in resolving this matter would be greatly appreciated.

![Screenshot (118)](https://github.com/thakurthegr8/loop/assets/93382253/5dfc8337-9f94-4237-be57-39c64fe4adca)

**Comment by thakurthegr8** [comment]
Make sure you configured right env variables @chayansurana3

**Comment by chayansurana3** [comment]
@thakurthegr8 
This is my '.env' file:
REACT_APP_FIREBASE_API_KEY=""
REACT_APP_FIREBASE_AUTH_DOMAIN=""
REACT_APP_FIREBASE_DATABASE_URL=""
REACT_APP_FIREBASE_PROJECT_ID=""
REACT_APP_FIREBASE_STORAGE_BUCKET=""
REACT_APP_FIREBASE_MESSAGING_SENDER_ID=""
REACT_APP_FIREBASE_APP_ID=""
REACT_APP_FIREBASE_MEASUREMENT_ID=""

Please recommend any changes.

**Comment by thakurthegr8** [comment]
Yeah .env variables are ok might mistyped some value @chayansurana3
